### PR TITLE
Rearrange training link

### PR DESCRIPTION
**Pull request by morshedniaz**
Rearrange the training link according to the xls file

### CODE REVIEW

**Comment by cosden** [comment]
@morshedniaz - looks good! Your git history is kind of a mess, but we'll squash it into a single commit to keep the main branch clean.

Here's a preview of what the site will look like when it goes live: https://49-293151248-gh.circle-artifacts.com/0/intersect-training.github.io/training-links/index.html. 

I like it and don't have any real concerns. A few of the categories now have just a single item in them...but I think that's ok. Other is now much smaller, which is good.

File: _data/training.yml [hunk 1]
```diff
@@ -145,65 +145,65 @@
...
   author: Nick Murphy
   description: This presentation discusses strategies for writing clean scientific software. This presentation encourages us to think of code as communication.
-  tags: {Development}
+  tags: {Software Engineering}
 - name: Python Packages
   url: https://py-pkgs.org/
   author: Tomas Beuzen & Tiffany Timbers
   description: Python Packages is an open source textbook that describes modern and efficient workflows for creating Python packages.
-  tags: {Packaging, Book}
+  tags: {Packaging, Pyhton in Research}
```
Comment (cosden) [inline _data/training.yml]:
```suggestion
  tags: {Packaging, Python in Research}
```

**Review by cosden — CHANGES_REQUESTED** [review]
One tiny little typo to fix. Otherwise looks good to me

**Comment by JeffCarver** [comment]
The changes look good to me. Hopefully we now have a structure that will allow us to do better sorting/filtering in the future.

**Comment by morshedniaz** [comment]
Dear Professor,

Thank you so much for your comments. I have corrected the spelling and tried to make another pull request on the same branch but I can't. Should I create another branch for the spelling correction? please let me know.

Sincerely yours
Niaz
Md Niaz Morshed
Graduate Research Assistant
Department of Computer Science
University of Alabama, Tuscaloosa
USA
________________________________
From: Ian Cosden ***@***.***>
Sent: Monday, January 10, 2022 2:25 PM
To: INTERSECT-training/intersect-training.github.io ***@***.***>
Cc: Md Niaz Morshed ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL] Re: [INTERSECT-training/intersect-training.github.io] Rearrange training link (PR #30)


@cosden requested changes on this pull request.

One tiny little typo to fix. Otherwise looks good to me

________________________________

In _data/training.yml<https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FINTERSECT-training%2Fintersect-training.github.io%2Fpull%2F30%23discussion_r781522131&data=04%7C01%7Cmmorshed%40crimson.ua.edu%7C3a3e6f5595f942b91c6808d9d4775757%7C2a00728ef0d040b4a4e8ce433f3fbca7%7C0%7C0%7C637774431293860613%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=ORuPXWkanj56Va%2Fu1DieBY7LjP%2BR3HdNCqfJXU9dVUQ%3D&reserved=0>:

>  - name: Python Packages

   url: https://py-pkgs.org/

   author: Tomas Beuzen & Tiffany Timbers

   description: Python Packages is an open source textbook that describes modern and efficient workflows for creating Python packages.

-  tags: {Packaging, Book}

+  tags: {Packaging, Pyhton in Research}



⬇️ Suggested change

-  tags: {Packaging, Pyhton in Research}

+  tags: {Packaging, Python in Research}



—
Reply to this email directly, view it on GitHub<https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FINTERSECT-training%2Fintersect-training.github.io%2Fpull%2F30%23pullrequestreview-848302657&data=04%7C01%7Cmmorshed%40crimson.ua.edu%7C3a3e6f5595f942b91c6808d9d4775757%7C2a00728ef0d040b4a4e8ce433f3fbca7%7C0%7C0%7C637774431293860613%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=33tFMURprAFBVPWKkRUT4%2FZb7balndM1%2BVZFOQkBZwQ%3D&reserved=0>, or unsubscribe<https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAVQXILHCNRC3WODH3AIK4MTUVM6DPANCNFSM5LQ5WBVQ&data=04%7C01%7Cmmorshed%40crimson.ua.edu%7C3a3e6f5595f942b91c6808d9d4775757%7C2a00728ef0d040b4a4e8ce433f3fbca7%7C0%7C0%7C637774431293860613%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=SfEbQhDiBWP18soUIyEmiSCon5RKeSIq%2FO%2FBgBvoLEE%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7Cmmorshed%40crimson.ua.edu%7C3a3e6f5595f942b91c6808d9d4775757%7C2a00728ef0d040b4a4e8ce433f3fbca7%7C0%7C0%7C637774431293860613%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=oE8eP22MAZI4f5WngIp43zNfw73DyW6PyLszSClf7vQ%3D&reserved=0> or Android<https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7Cmmorshed%40crimson.ua.edu%7C3a3e6f5595f942b91c6808d9d4775757%7C2a00728ef0d040b4a4e8ce433f3fbca7%7C0%7C0%7C637774431293860613%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=78h9svZmXtofeDYL1%2BUiIr25uCyaLjg5iX67jmlFPN4%3D&reserved=0>.

**Comment by cosden** [comment]
Your latest commit fixed it (https://github.com/INTERSECT-training/intersect-training.github.io/pull/30/commits/648a19cd70ad6bac3b4a5dbdc2da6bd1c7f061af). 

Looks good. Thanks!

**Review by cosden — APPROVED** [review]

**Pull request merged**